bokehjs/test/integration/axes/with_math_text.ts

```diff
@@ -0,0 +1,75 @@
+import sinon from "sinon"
+
+import {MathTextView, TeX} from "@bokehjs/models/text/math_text"
+import {
+  Plot,
+  Range1d,
+  LogScale,
+  LogAxis,
+} from "@bokehjs/models"
+
+import {display, InternalProvider} from "../_util"
+
+describe("MathText on axes", () => {
+  const tex = "x = \\frac{-b \\pm \\sqrt{b^2 - 4ac}}{2a}"
+
+  type PlotFn = (
+    attrs: Partial<LogAxis.Attrs>
+  ) => Promise<void>
```

Comment:
No need to split types like this so aggressively. It's a single argument function, just keep it on a single line.